STYLE-GUIDE.md

```diff
@@ -0,0 +1,22 @@
+# Exercism's Style Guide
+
+This document acts as a Style Guide for the language and wording used in exercises.
+
+## General principle
+
+All content should be written in US english. 
```

Comment:
```suggestion
All content should be written in US English. 
```

Reply:
Maybe:

```
Regrettably, all content should be written in Simple English, colloquially known as US English.
```

Just kidding of course. But we might want to consider linking to a document that has the most prominent differences between English and US English, as they are **not** apparent to everyone.

Reply:
I remember there being discussions on Slack about the use of `pants` vs `trousers` where @iHiD veto'd the use of US English. Perhaps that should be added as an exception to this rule? 🙃

Reply:
I'm happy to have "no pants" as a documented rule!

Reply:
Dogs have pants, people have trousers.

Reply:
https://github.com/exercism/problem-specifications/issues/1715